charts/rasa-x/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 
-version: "1.4.1"
+version: "1.4.2"
```

Comment:
I think we should bump to `1.5` as we are changing dependencies, wdyt?

Reply:
Good point